javascript/javascript3/week3/screenshot/screenshot.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="screenshot.css">
+    <title>Screen shot</title>
+</head>
+
+<body>
+    <main>
+        <h1 class="heading-1">Get a Screenshot</h1>
+        <form id="create-new-screenshot">
+            <fieldset>
+                <legend>Screen-Shot-Generate And Saved</legend>
+                <div class="form-field">
+                    <label class="label" for="add-screenshot">Screen-Shot-url*</label>
+                    <input type="text" class="input" id="add-screenshot" />
```

Comment:
```suggestion
                    <input type="text" class="input" id="add-screenshot" required/>
```
Good reading https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation

Reply:
I will implement this,actually our homework was divided into 3parts and this part was in extra extra part ,I tried to focus on logic how could I implement that extra part and due to that time validation skipped from mind ,thank you for pointing this ,I will fix this .